design/proposals/load_policy/load_policy_validation.md

```diff
@@ -0,0 +1,285 @@
+
+Load policy with empty value,  policy validation is not activated, Same error as previous version
+
+Load policy with just policy id,  policy validation is not activated, Same error as previous version
+
+#### Functionality 
+Load policy with 1 host with JWT annotations, policy validation is  activated with expected output
+* Write both success and failure scenarios 
+
+Load nested policies with 5 hosts with JWT annotations, policy validation is  activated with expected output
+* Write both success and failure scenarios 
+
+Load hosts with and without JWT annotations in same policy,  policy validation is  activated with expected output
+* Write both success and failure scenarios 
+
+#### Performance
+Run performance tests of load policy before and after the feature with and without JWT hosts 
+
+#### Load
+Load policy with 100 host with JWT annotations, policy validation is  activated with expected output
+
+### Out of Scope
+* This design do not handled the upgrade process and how to migrate invalid hosts from 
+old version to new version.
+
+## Documentation
+* We will need to document the UX for all kinds of put/post/patch in case of changing/not changing
+the DB, since the validation occurs only on the DB final state.
+  So there are possible use cases were the policy input will be invalid 
+  (For example: empty annotations), but the validation will passed successfully since the DB may not changed.
+
+## Security
+TBD
+
+## Infrastructure
+None.
+
+## Audit
+* No need for new audit records 
+
+## Open Questions
+None.
+
+## Delivery Plan
+
+### 1st plan - 1 milestone Total EE 45
+
+![load-policy-m1-task-dependencies](load-policy-m1-task-dependencies.png)
+
+#### Ramp-up EE 2
+* (Optional) Learn JWT authenticator code + load policy flow **EE 2**
+
+#### Designs EE 2
+* How to refactor JWT authentication to use new validation logic in execute the same code in both flows **EE 1**
+
+#### Implementations EE 19
+* Write the integration point in load policy Skeleton and empty classes according to HLD **EE 3**
+* Write GetJWTHosts command **class EE 2**
+* Write ValidateJwtHostConstraints command class **EE 2**
+* Implement JWT authentication refactor to use ValidateJwtHostConstraints command class **EE 3**
+* Implement CreateHostValidatorsFactory command class **EE 2**
+* Implement ValidateHostConstraints command class **EE 2**
+* Implement CreatePolicyEntityValidatorsFactory command class **EE 2**
+* Implement ValidatePolicyConstraints command class **EE 2**
+* Enhance logs and supportability **EE 1**
+
+#### Testing EE 13
+* Implement Conjur integration tests according to test plan **EE 10**
+* Implement Appliance integration tests according to test plan **EE 3**
+
+#### Security EE 5
+* Review and actions **EE 5**
+
+#### Docs EE 3
+
+#### Demo EE 1
+
+### 2nd plan - 2 milestones Total EE 45
+
+#### M1 - Load policy infrastructure  SP 19
+
+**Implementations** - 16 SP
+* No need of 3.d
```

Comment:
impl - 16SP
the bullet says 'no need' and 3d
what needs to be done? why 16? why 3?

Reply:
this is the implementation tasks from plan 1 without 3.d task